sdk/api-learn/Azure.Learn.AppConfig/src/Azure.Learn.AppConfig.csproj

```diff
@@ -0,0 +1,33 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Description></Description>
+    <AssemblyTitle></AssemblyTitle>
+    <Version></Version>
+    <PackageTags></PackageTags>
+    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+    <NoWarn>CS1591;AZC0012;CA1308</NoWarn>
```

Comment:
Do we have to keep AZC0012 supressed?

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/14303